tagmap/generate_tagset_and_conversion_map.py

```diff
@@ -110,93 +33,47 @@ def prepare_structurized_data():
     #     {tags: [other tags], card: cardinality}
     # )
     # }
-    structurized_data = defaultdict(list)
+    structured_data = defaultdict(list)
     for s in sets:
         flexeme = s[0]
-        flexeme_data = FlexemeSubclass(tags=s[1], card=s[2])
-        structurized_data[flexeme] += [flexeme_data]
+        flexeme_data = dict(tags=s[1], card=s[2])
+        structured_data[flexeme] += [flexeme_data]
 
-    return structurized_data
+    return structured_data, set(tags)
 
 
 @click.command(help='Generate tagset and nkjp2us mapping')
 @click.option("--tagset-filepath", type=str, default="./data/tagmap_data/transitional_tagset.json")
 @click.option("--conversion-map-filepath", type=str, default="./data/tagmap_data/nkjp2us.json")
 @click.option("--min-cardinality", type=int, default=100)
+@click.option("--strategy", type=str, default='justpos')
```

Comment:
Wywaliłbym domyślną strategię - dzięki temu że zawsze trzeba ją podać, to stepy dvc będą lepiej dokumentować to co dzieje się po kolei